pkg/cmd/pr/status/http.go

```diff
@@ -189,7 +189,7 @@ func pullRequestStatus(httpClient *http.Client, repo ghrepo.Interface, options r
 func pullRequestFragment(httpClient *http.Client, hostname string) (string, error) {
 	fields := []string{
 		"number", "title", "state", "url", "isDraft", "isCrossRepository",
-		"headRefName", "headRepositoryOwner", "mergeStateStatus",
+		"headRefName", "headRepositoryOwner", "mergeable", "mergeStateStatus",
```

Comment:
Requesting the `mergeable` field kicks off a potentially expensive (and sometimes async) job of calculating mergeability of a PR. Requesting this field on all records when fetching PRs in bulk might have negative consequences. Do you have thoughts about that?

Reply:
@mislav I'll check with my team about this concern and see if any of the experts have concerns or can suggest an alternative.

Reply:
we're making internal changes that will improve the cost of this operation. let's revisit at the end of august and see where things are with that work

Reply:
The merge performance operations rolled out (details[^too] https://github.com/github/pull-requests/issues/4278):

<img width="1000" alt="merge-ort perf improvements" src="https://user-images.githubusercontent.com/7074/189169400-4711bf19-9b54-414d-a79d-5e63eca8ebdb.png">

[^too]: Too many details.